tox.ini

```diff
@@ -12,9 +12,11 @@ install_command =
   pip install {opts} {packages}
 deps =
   mypy
-  stestr
```

Comment:
Whoops actually... Nix package relies on `stestr` in `postCheck`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/version-management/git-and-tools/git-machete/default.nix#L27

TBD how to handle the change smoothly

Reply:
Okay, leaving .stestr.conf as for now; I'll clean up all stestr-related stuff once I've switched Nix package definition to pytest